src/protocolMessaging/index.js

```diff
@@ -261,6 +261,10 @@ class Router {
       ...order,
       v: order.v ? ethers.utils.bigNumberify(order.v).toNumber() : order.v,
       expiration: order.expiration ? ethers.utils.bigNumberify(order.expiration).toNumber() : order.expiration,
+      makerAddress: (order.makerAddress || '').toLowerCase(), // normalizes the case of addresses in returned orders
```

Comment:
would this invalidate the ECDSA signatures if the maker signed without lower casing?

Reply:
No was able to fill an order before and after altering the case, I think because it's fundamentally the same number
<img width="89" alt="Screen Shot 2019-06-04 at 11 19 16 AM" src="https://user-images.githubusercontent.com/1197232/58891509-a2825400-86ba-11e9-84fa-9fcd28ae2801.png">